Script/owterra.dat

```diff
@@ -150,7 +150,7 @@ pyramid
   AttachedDungeon = PYRAMID;
   CanBeGenerated = true;
   HideLocationInitially = true;
-  NativeGTerrainTypes = { 1, DESERT; }
+  NativeGTerrainTypes = { 2, DESERT, JUNGLE; }
```

Comment:
@red-kangaroo would it be acceptable if the pyramid has the flexibility to generate in either desert or a jungle tile? The generator would try to place the pyramid on a desert tiles first, as a matter of preference. On a minority of tiny sized worlds, the sampler might not pick up any desert tiles, so this change gives the generator the option to fall back to jungle terrain. This means the generator does not need to generate as many worlds. Like, ~10 worlds instead of ~100-1000 :)

Reply:
Sure, no problem. :)